### PR TITLE
Improve wording of html template tag and hyperscript description

### PR DESCRIPTION
**Pull request by trusktr**
This removes some sarcasm that can appear to be negative. After all, no one likes less performance on purpose, so this explains the downsides in a more neutral and welcoming way.

### CODE REVIEW

**Comment by ryansolid** [comment]
We shouldn't lose the fact it means manually wrapping expressions(very important) and larger bundle size. Performance is not what I cared about when writing this. The DX is significantly compromized that I think this should be discouraged. I've gone through this enough with people on the Mithril side of things I can't imagine trying to explain this to unsuspecting people who might think this is an equivalent experience to JSX.

**Comment by trusktr** [comment]
Oh yes, my bad. It's late, getting zzzzzz, I'll be back (not in The Terminator voice)

**Comment by trusktr** [comment]
Updated. This explains what the words actually mean now. Previously, it said "manual work to wrap expressions" but the example didn't show that, so people, especially newcomers, wouldn't know what it is referring to.

File: langs/en/guides/getting-started.md [hunk 1]
```diff
@@ -109,11 +109,35 @@ You can run them straight from the browser using [Skypack](https://www.skypack.d
...
+  html`
+    <h1>Hello ${() => first() + ' ' + last()}</h1>
+  `
+  // or
+  h('h1', {}, 'Hello ', () => first() + ' ' + last())
+  ```
+  Solid's JSX doesn't have this issue because of its compile-time abilities, and an expression like `<h1>Hello {first() + ' ' + last()}</h1>` will be reactive.
+- Build-time optimizations won't be in place, meaning app startup speed will be slightly slower because each template gets compiled at runtime the first time it is executed, but for many use cases this perf hit is imperceivable. Ongoing speed after startup will remain the same with the ``` html`` ``` template tag as with JSX, while `h()` will always have slower ongoing speed due to its inability to statically analyze a whole template before it is executed.
+
+You need the corresponding DOM Expressions library for these to work with TypeScript. You can use Tagged Template Literals with [Lit DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/lit-dom-expressions) or HyperScript with [Hyper DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/hyper-dom-expressions).
```
Comment (trusktr) [inline langs/en/guides/getting-started.md:143]: Is this last paragraph still needed? I haven't checked. But we can include types as part of `/html` or `/h` without requiring extra packages.

**Comment by Jutanium** [comment]
Awesome, I'll keep all this in mind on the new docs site too.

**Pull request merged**